components/admin_console/admin_sidebar/admin_sidebar_section.tsx

```diff
@@ -74,9 +72,7 @@ const AdminSidebarSection = ({name, title, action, children = [], definitionKey,
 
     if (type === 'text') {
         sidebarItem = (
-            <div
-                className={`${className}-title`}
-            >
+            <div className={`${className}-title`}>
                 <span className={`${className}-title__text`}>
```

Comment:
nit: could we eliminate the `span`, since using it only to style the complete content of a div is a bit over the top. It also helps to remove the DOM clutter a bit.